spring-integration-mqtt/src/test/java/org/springframework/integration/mqtt/Mqttv5BackToBackAutomaticReconnectTests.java

```diff
@@ -104,7 +92,7 @@ public static class Config {
 		@Bean
 		public MqttConnectionOptions mqttConnectOptions() {
 			return new MqttConnectionOptionsBuilder()
-					.serverURI("wss://badMqttUrl")
+					.serverURI(MosquittoContainerTest.mqttUrl())
```

Comment:
There was really the point to have this test exactly this way.
Please, see my comment about remove `reconnect()` an related issue when that feature and this test have need introduced.
And consider to reinstate that logic and this test.

Thank you!